packages/backend/src/Actions/deleteActivity.js

```diff
@@ -0,0 +1,8 @@
+export const DELETE_ACTIVITY = 'DIRECT_LINE/DELETE_ACTIVITY';
+
+export default function (activityID) {
```

Comment:
some template for these?

Reply:
Will stay away from [Action Creators](https://redux.js.org/recipes/reducingboilerplate#generating-action-creators) for simplicity because action creators are not a built-in function from Redux.